src/main/kotlin/no/nav/syfo/dialogmote/DialogmoteService.kt

```diff
@@ -190,18 +173,23 @@ class DialogmoteService(
                 )
                 createAndSendVarsel(
                     connection = connection,
-                    arbeidstakerId = createdDialogmoteIdentifiers.motedeltakerArbeidstakerIdList.first,
-                    arbeidstakerUuid = createdDialogmoteIdentifiers.motedeltakerArbeidstakerIdList.second,
-                    arbeidsgiverId = createdDialogmoteIdentifiers.motedeltakerArbeidsgiverIdList.first,
+                    arbeidstakerId = createdDialogmoteIdentifiers.motedeltakerArbeidstakerIdPair.first,
```

Comment:
Begynner å bli en del parametere til denne createAndSendVarsel-metoden nå, moden for en refaktorering?
Kanskje også noe i denne servicen kunne vært flyttet til en DialogmoteVarselService f.eks.
I egen PR 😉

Reply:
Ja, sikkert noe forbedringspotensiale her. Kan vurdere refaktorering i egen PR ja.